lib/fileutils.rb

```diff
@@ -751,7 +757,7 @@ def link_entry(src, dest, dereference_root = false, remove_destination = false)
   end
   module_function :link_entry
 
-  # Copies files.
+  # Copies files; returns +nil+.
```

Comment:
This method doesn't always return `nil`. It will return `nil` when we don't pass in `preserve`. But it will return an integer when `preserve: true`.

```ruby
irb(main):005:0> FileUtils.cp("testing", "foo", preserve: true)
=> 1
```

For this reason, the return values of `FileUtils` methods are not significant and I prefer if we didn't specify what their return values are.